packages/@aws-cdk/aws-dynamodb/lib/table-time-to-live-handler/utils.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-console */
+import type { OnEventRequest } from '@aws-cdk/custom-resources/lib/provider-framework/types';
+import { AWSError, DynamoDB } from 'aws-sdk'; // eslint-disable-line import/no-extraneous-dependencies
+const dynamodb = new DynamoDB({ apiVersion: '2012-08-10' });
+
+
+export async function disableTimeToLive(event: OnEventRequest) {
+  const ttl = await dynamodb.describeTimeToLive({ TableName: event.ResourceProperties.TableName }).promise();
+  await dynamodb.updateTimeToLive({
+    TableName: event.ResourceProperties.TableName,
+    TimeToLiveSpecification: {
+      AttributeName: ttl.TimeToLiveDescription?.AttributeName ?? '',
```

Comment:
Can't we get the `AttributeName` from the `event`?

Reply:
Yes, we could get that from the event. Doing it this way however allows to disable the ttl even when it was previously set outside of the stack (e.g., via the aws console). I thought this behavior would be nice to have, but I suppose it is up to whether this scenario is within the scope of the CDK or not.

Reply:
Getting it from `OldResourceProperties` would better mimic the CF behavior I think.

Reply:
Yeah, let's get it from `OldResourceProperties`. Saves us an API call as well.

Reply:
I have updated the function accordingly and it now uses `OldResourceProperties.TimeToLiveAttribute`.